net/ntpd/Makefile

```diff
@@ -19,7 +19,6 @@ PKG_LICENSE:=NTP
 PKG_LICENSE_FILES:=COPYRIGHT html/copyright.html
 PKG_CPE_ID:=cpe:/a:ntp:ntp
 
-PKG_FIXUP:=autoreconf
```

Comment:
@neheb Sorry, I've missed this pull request, just noticed it via the closure of #24918, so take this as a kind of post review :)

IMO this is not a proper fix, but rather a workaround. There is probably something wrong with the autofoo (or the package in general) and should be investigated and fixed by other means.

In other words, the `PKG_FIXUP:=autoreconf` is correct, probably should be default for all packages, because we certainly dont want to use autofoo as being shipped, but generate those by our tools/versions.

Reply:
Additionally I would like to point out, that those autofoo fixes were backported into 23.05 https://github.com/openwrt/openwrt/commit/d811a09524118451d14a0ae8abce21fc123a3bf0 and https://github.com/openwrt/openwrt/commit/c4c43c64aa6c5ba44322ddefa106b9e707b64781, thus we're looking for some ntpd package fix which might be backported there as well.

Reply:
Is there any better solution for now? Currently, this package is not compiled in OpenWrt 23.05 and it would be great to be fixed there.

Reply:
> Is there any better solution for now?

Not yet https://github.com/openwrt/openwrt/pull/15825#issuecomment-2342047783, but likely the autofoo in the package would need to be patched.

> this package is not compiled in OpenWrt 23.05

You mean when compiled in the tree? I don't see that package in the [faillogs-23.05](https://downloads.openwrt.org/releases/faillogs-23.05/), but I've checked just few architectures.